src/Kunstmaan/NodeBundle/Helper/Services/PageCreatorService.php

```diff
@@ -95,118 +120,168 @@ public function setContainer(ContainerInterface $container = null)
      *
      * @throws \InvalidArgumentException
      */
-    public function createPage(HasNodeInterface $pageTypeInstance, array $translations, array $options = array())
+    public function createPage(HasNodeInterface $pageTypeInstance, array $translations, array $options = [])
     {
-        if (is_null($options)) {
-            $options = array();
-        }
+        Assert::notEmpty($translations, 'There has to be at least 1 translation in the translations array');
+        $this->pageTypeInstance = $pageTypeInstance;
+        $this->options = (is_null($options)) ? [] : $options;
```

Comment:
```$this->options = (is_null($options)) ? [] : $options;```
Maybe to
```$this->options = null === $options ? [] : $options;```

Reply:
Yeah sure, although I suggest we keep the brackets around the boolean condition (ternary readability trick). 
`$this->options = (null === $options) ? [] : $options;`

Reply:
Correct :)

Reply:
Did you change it back to is_null on purpose?